kafka/metrics.go

```diff
@@ -60,11 +70,31 @@ type metricHooks struct {
 	namespace   string
 	topicPrefix string
 
-	messageProduced          metric.Int64Counter
-	messageFetched           metric.Int64Counter
-	messageDelay             metric.Float64Histogram
-	messageBytes             metric.Int64Counter
-	messageUncompressedBytes metric.Int64Counter
+	// kotel metrics
+
+	connects    metric.Int64Counter
+	connectErrs metric.Int64Counter
```

Comment:
question: as we submitted a PR to update these metrics (https://github.com/twmb/franz-go/pull/691) do you think it make sense to keep compatibility with `kotel` here? I don't see benefits in the long term but I'm ok if we want to keep compatibility for this PR

Reply:
FWIW I plan to merge 691 actually sooner than later, if that's the problem.